components/pkg-export-docker/src/error.rs

```diff
@@ -28,11 +31,15 @@ pub type Result<T> = result::Result<T, Error>;
 pub enum Error {
     BuildFailed(ExitStatus),
     DockerImageIdNotFound(String),
+    ECRTokenParseError(DecodeError),
+    Base64DecodeError(FromUtf8Error),
```

Comment:
I think the naming of these is a little confusing.  I would expect the `Base64DecodeError` to be the thing that you use when `base64::decode` fails.  We have that pattern in several other crates, and I think it reads a bit better (principle of least surprise, etc)